pkg/ddevapp/app_compose_template.yaml

```diff
@@ -203,6 +204,7 @@ services:
     - TZ={{ .Timezone }}
     - USER={{ .Username }}
     - VIRTUAL_HOST=${DDEV_HOSTNAME}
+    - YARN_CACHE_DIR=/mnt/ddev-global-cache/yarn/${DDEV_PROJECT}-web
```

Comment:
it's `YARN_CACHE_FOLDER`, otherwise it fails when you run ddev start, I added a dependency to my repo.

Reply:
Sorry, it fails when you run `ddev yarn install`, not when you start it.

Reply:
```
Usage Error: Unrecognized or legacy configuration settings found: cacheDir - run "yarn config -v" to see the list of settings supported in Yarn (in <environment>)
```

Reply:
```suggestion
    - YARN_CACHE_FOLDER =/mnt/ddev-global-cache/yarn/${DDEV_PROJECT}-web
```

Reply:
BTW, the suggestion technique above saves everybody time and trouble. You'll like it. Anybody can do it. Then I can just commit it.